sql/updates/world/3.3.5/9999_99_99_99_AZJOLNERUB.sql

```diff
@@ -0,0 +1,16 @@
+-- ------------------------- --
+-- Krik'thir the Gatewatcher --
+-- ------------------------- --
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=13 AND `SourceEntry` IN (52438,52449);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`,`SourceGroup`,`SourceEntry`,`SourceId`,`ElseGroup`,`ConditionTypeOrReference`,`ConditionTarget`,`ConditionValue1`,`ConditionValue2`,`ConditionValue3`,`NegativeCondition`,`Comment`) VALUES
+(13,1,52438,0,0,31,0,3,22515,0,0,"Summon Skittering Swarmer - Target World Trigger"),
+(13,1,52449,0,0,31,0,3,22515,0,0,"Summon Skittering Infector - Target World Trigger");
+
+-- Anub'ar Skirmisher: Fixtate
+DELETE FROM `spell_script_names` WHERE `scriptname` = "spell_anub_ar_skirmisher_fixtate";
+INSERT INTO `spell_script_names` (`spell_id`,`scriptname`) VALUES (52536,"spell_anub_ar_skirmisher_fixtate");
+
+-- Watchers: Web Wrap 
+DELETE FROM `spell_script_names` WHERE `scriptname` = "spell_gatewatcher_web_wrap";
+INSERT INTO `spell_script_names` (`spell_id`,`scriptname`) VALUES (52086,"spell_gatewatcher_web_wrap");
+UPDATE `creature_template` SET `AIName`='',`ScriptName`="npc_gatewatcher_web_wrap" WHERE `entry`=28619;
```

Comment:
Fixed after comment above (but comment was after the EOF and was not hidden by the update).